### PR TITLE
Fix test workflow

### PR DESCRIPTION
**Pull request by szepeviktor**
A better workflow name would be "Integrate" as i in CI.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -9,24 +9,22 @@ on:
 
 jobs:
   test:
+    name: Tests on ${{ matrix.python-version }}
```
Comment (szepeviktor) [inline .github/workflows/test.yml:12]: Display Python version in job name

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -9,24 +9,22 @@ on:
...
     runs-on: ubuntu-latest
     strategy:
-      max-parallel: 2
```
Comment (szepeviktor) [inline .github/workflows/test.yml:14]: # 👩🏻‍🦳🧊🤍
Let it go!
🇵🇹 https://www.youtube.com/watch?v=sWw71mv8uKs

**Comment by mnot** [comment]
Thanks!

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**